packages/site-demo/content/product/components/thumbnail/index.mdx

```diff
@@ -34,9 +34,26 @@ Thumbnails come in two variants: `squared` and `rounded`
 
 <DemoBlock orientation="horizontal" demo="combined" />
 
-## Loading and error states
+## Loading skeleton
 
-<DemoBlock orientation="horizontal" demo="loading-error" withThemeSwitcher />
+<DemoBlock orientation="horizontal" demo="loading" withThemeSwitcher />
+
+While loading the image, the thumbnail displays a loading skeleton but only if the thumbnail has sufficient
+configuration to know what size to take. To do that, you can do any of the following:
+
+- use a fixed ratio (any ratio except `free` and `original`) and a width constraint (using `size` or manual CSS).
+- use `imgProps={{ width, height }}` where `width` and `height` is the original image size (or same original size ratio).
+  the thumbnail will scale down if the given `width` and `height` are bigger than allowed space
+
+## Error state
+
+If the image loading fails, the thumbnail will display an error state by default.
+
+<DemoBlock orientation="horizontal" demo="error" withThemeSwitcher />
+
+You can customize this state using the `fallback` prop to change the icon or even replace with a custom React node.
```

Comment:
Error state can be customize using the `fallback` prop to change the icon or even replace with a custom React node.